packages/apputils-extension/src/index.ts

```diff
@@ -650,7 +650,7 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
     });
 
     commands.addCommand(CommandIDs.displayShortcuts, {
-      label: trans.__('Show Keyboard Shortcuts'),
+      label: trans.__('Show Keyboard Shortcuts...'),
```

Comment:
This looks to have been carried over from another PR. Let's revert it and snapshots which changed as a consequence of this change.

```suggestion
      label: trans.__('Show Keyboard Shortcuts'),
```